docs/framework/tools/al-exe-assembly-linker.md

```diff
@@ -26,160 +26,166 @@ ms.author: "mairaw"
-|al1072|Assembly and module 'Module Name' cannot target different processors<br /><br /> You cannot link an assembly and a module that are targeted for different processors, because the result has to run on a single processor.|  
-|al1073|Referenced assembly 'assembly' targets a different processor<br /><br /> You cannot link assemblies that are targeted for different processors, because the result has to run on a single processor.|  
-|al1074|Module name 'Module Name' stored in 'File Name' must match its file name<br /><br /> This is required of the linker. To resolve this problem, make the two names match.|  
-|al1075|Delay signing was requested, but no key was given<br /><br /> When an assembly is delay signed, the compiler does not compute and store the signature, but reserves space in the file so the signature can be added later.<br /><br /> For example, using **/delaysign+** enables a tester to put the assembly in the global cache. After testing, you can fully sign the assembly by adding the private key to the assembly by using the Assembly Linker utility.|  
-|al1076|Type 'type' is forwarded to multiple assemblies: 'assembly' and 'assembly'.<br /><br /> A type can only be forwarded to one assembly.|  
-|al1077|Public type 'type' is defined in 'assembly' and forwarded to 'assembly'.<br /><br /> There is a duplicate public type in the assembly being generated. One is a valid type definition and the other is a type forwarder.|  
-  
-## Example  
- The following command creates an executable file `t2a.exe` with an assembly from the `t2.netmodule` module. The entry point is the `Main` method in `MyClass`.  
-  
-```  
-al t2.netmodule /target:exe /out:t2a.exe /main:MyClass.Main  
-```  
-  
-## See Also  
- [Tools](../../../docs/framework/tools/index.md)   
- [Sn.exe (Strong Name Tool)](../../../docs/framework/tools/sn-exe-strong-name-tool.md)   
- [Gacutil.exe (Global Assembly Cache Tool)](../../../docs/framework/tools/gacutil-exe-gac-tool.md)   
- [Programming with Assemblies](../../../docs/framework/app-domains/programming-with-assemblies.md)   
- [Command Prompts](../../../docs/framework/tools/developer-command-prompt-for-vs.md)
+> To avoid being subject to virtualization on a Windows Vista computer, your assembly must include a Win32 manifest that specifies the requested execution level. When you use *Al.exe* directly from the command line, you can either embed the manifest in a Win32 resource file or use *mt.exe* to append the manifest at a later stage in the build process. Starting with [!INCLUDE[vs_orcas_long](../../../includes/vs-orcas-long-md.md)], both the C# and Visual Basic compilers automatically embed a Win32 manifest into the assembly. For more information, see [/win32manifest (C# Compiler Options)](~/docs/csharp/language-reference/compiler-options/win32manifest-compiler-option.md).
```

Comment:
Since Vista is not supported on later .NET Framework versions, you can delete everything up to the "Starting with..."